ingest-app/src/cmr/ingest/api/routes.clj

```diff
@@ -250,13 +251,33 @@
            (bulk/get-provider-tasks :granule provider-id request)))))))
 
 ;; TODO: Generic work - should this change to be more dynamic, gen type in front?
+
 (def generic-document-routes
   (context "/generics/provider/:provider-id/:native-id" [provider-id native-id]
     (POST "/" request (gen-doc/create-generic-document request))
     (GET "/" request (gen-doc/read-generic-document request))
     (PUT "/" request (gen-doc/update-generic-document request))
     (DELETE "/" request (gen-doc/delete-generic-document request))))
 
+(def generate-generic-concept-types-reg-ex
+  "This function creates a regular expresion for all of the generic concepts.  This is used to create the api endpoints.
+   An example string that is return looks like: \"dataqualitysummary|orderoption|serviceoption\" "
+  (let [x (-> (str (concepts/get-generic-concept-types-array))
```

Comment:
Can we change the name from `x` to `rx`?

Reply:
yes, I have made the change.